pom.xml

```diff
@@ -584,14 +589,15 @@
         <imp.pkg.version.javax.servlet>[2.6.0, 3.0.0)</imp.pkg.version.javax.servlet>
         <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
         <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
+        <identity.data.publisher.authentication.version>5.1.9-SNAPSHOT</identity.data.publisher.authentication.version>
```

Comment:
remove snapshot dependency

Reply:
Merge wso2-extensions/identity-data-publisher-authentication#60 for update the dependency with release version